.github/cloud-samples-tools/pkg/utils/affected.go

```diff
@@ -0,0 +1,53 @@
+/*
+ Copyright 2024 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package utils
+
+import (
+	"slices"
+)
+
+func Affected(config Config, diffs []string) ([]string, error) {
+	changed := Changed(config, diffs)
+	if slices.Contains(changed, ".") {
```

Comment:
Wow is this new fancy `slices` packages nice. Back in my day ... ;D.

[No action required]